TicTacToe-app/src/app/shared/services/api.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed, inject } from '@angular/core/testing';
```

Comment:
services should be tested in the beginning since these are the easiest to test. We can ignore component unit test for now